introduction.md

```diff
@@ -1,2 +1,4 @@
 I am Pragnesh and this is Project Introduction file.
-My favorite codeschool path : https://www.codeschool.com/paths/javascript
\ No newline at end of file
+My favorite codeschool path : https://www.codeschool.com/paths/javascript
+*Ruby
+*Javascript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?